recipes/treesapp/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "treesapp" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2cbce8aaab07e5b493f231b6af75724def82e5785f3b7f13f101c92d49d8550
+
+build:
+  number: 0
+  skip: True  # [not linux64]
+  entry_points:
+    - treesapp = treesapp.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python >=3.5
+  host:
+    - biopython >=1.68
```

Comment:
Most of these dependencies should probably be just `run` dependencies. I suggest these changes:
- Remove `python` from build dependencies
- Leave these as the only host dependencies: pip, python